qcengine/programs/empirical_dispersion_resources.py

```diff
@@ -649,7 +649,7 @@
             "pbe": {"params": {"s6": 1.000, "s8": 0.358940, "a1": 0.012092, "a2": 5.938951}},
             "pbe0": {"params": {"s6": 1.000, "s8": 0.528823, "a1": 0.007912, "a2": 6.162326}},
             "wpbe": {"params": {"s6": 1.000, "s8": 0.906564, "a1": 0.563761, "a2": 3.593680}},  # formerly lcwpbe
-            "sapt0": {"params": {"s6": 1.000, "s8": 1.409027, "a1": 0.078803, "a2": 4.553948}},  # experimental
+            "sapt0": {"params": {"s6": 1.000, "s8": 1.03220439, "a1": 0.00704073, "a2": 6.11845609}},  # experimental
```

Comment:
how's the refitting going, @dsirianni? note that 4-6 sig figs is plenty, so perhaps the procedure is optimize, truncate, recompute to make sure we're using the same tidy parameter values here and in paper.